src/main/scanLogic/scanManager.ts

```diff
@@ -131,7 +131,7 @@ export class ScanManager implements ExtensionComponent {
     private getResources(supportedScans: SupportedScans): Resource[] {
         let resources: Resource[] = [];
         if (supportedScans.applicability || supportedScans.iac || supportedScans.secrets) {
-            resources.push(BinaryRunner.getAnalyzerManagerResource(this._logManager));
+            resources.push(BinaryRunner.getAnalyzerManagerResource(this._logManager, undefined));
```

Comment:
I think "undefined" is the default when a parameters is not provided
```suggestion
            resources.push(BinaryRunner.getAnalyzerManagerResource(this._logManager));
```